src/hooks/useCombobox/__tests__/memo.test.js

```diff
@@ -0,0 +1,9 @@
+import {renderUseCombobox} from '../testUtils'
+
+test('functions are memoized', () => {
```

Comment:
This is just a simple test to make sure nothing is changed between rerenders when nothing has changed 🙃

Before my changes, this test would fail.